source/tabs/sockets.rst

```diff
@@ -0,0 +1,62 @@
+Sockets
+=======
+
+Sockets are the list of physical sockets present on the hardware. These sockets can be Ethernet, USB, HDMI, etc.
+This information cannot be returned by the automatic inventory, so you have to add it manually.
+
+It enables hardware to be linked by cables. Socket is also linked to the :doc:`cables <../modules/assets/cables>` object
+
+You can add them either en masse or one by one.
+
+.. image:: ../../image/sockets.png
+   :alt: Networl port locks
+   :align: center
+   :scale: 49%
+
+Add a socket
+------------
+
+* To add a socket, enter a name,
+* Select the socket model. You can add them by clicking on **+**.
+.. image:: ../../image/sockets-add-socket-model.png
+   :alt: Networl port locks

```

Comment:
typo but wrong alt I think:
```suggestion
   :alt: Add socket model
```

Reply:
The typo was fixed before a submit my review, but I still think "alt" content is wrong.